lightning/src/ln/channelmonitor.rs

```diff
@@ -445,72 +445,63 @@ struct LocalSignedTx {
 	htlc_outputs: Vec<(HTLCOutputInCommitment, Option<Signature>, Option<HTLCSource>)>,
 }
 
-#[derive(PartialEq)]
-enum InputDescriptors {
-	RevokedOfferedHTLC,
-	RevokedReceivedHTLC,
-	OfferedHTLC,
-	ReceivedHTLC,
-	RevokedOutput, // either a revoked to_local output on commitment tx, a revoked HTLC-Timeout output or a revoked HTLC-Success output
-}
-
 /// When ChannelMonitor discovers an onchain outpoint being a step of a channel and that it needs
 /// to generate a tx to push channel state forward, we cache outpoint-solving tx material to build
 /// a new bumped one in case of lenghty confirmation delay
 #[derive(Clone, PartialEq)]
-enum InputMaterial {
+pub(crate) enum InputMaterial {
 	Revoked {
-		script: Script,
+		witness_script: Script,
 		pubkey: Option<PublicKey>,
 		key: SecretKey,
 		is_htlc: bool,
-		amount: u64,
+		revoked_amount: u64,
```

Comment:
I really dont understand the need for this commit - amount, revoked_amount, local_amount and remote_amount are all the amount as it appears in the transaction output. I think the new names may be even *more* confusing as it'll make me do a double-take to make sure we don't need to calculate them from some Channel data instead of from on-chain outputs.

Reply:
Okay was in reaction to https://github.com/rust-bitcoin/rust-lightning/pull/462#discussion_r379195303, I'm ~0 on this, dropped the commit